src/validation.jl

```diff
@@ -166,6 +166,15 @@ function check_limits(
     return check_limits_impl(valid_info, field_value)
 end
 
+function check_limits(
```

Comment:
Why this and not just changing the function in line 160 to take `Union{Nothing, Real}` ?

Reply:
Good point

Reply:
I agree, makes more sense to do that